4-Express/server/mychecklist_server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const cors = require('cors')
+const app = express()
+const port = 3000
+
+const items = {};
+
+app.use(cors());
+app.use(
+  express.urlencoded({
+    extended: true
+  })
+);
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.sendJson(items);
+})
+
+app.post('/:id', (req, res) => {
```

Comment:
The REST api convention for saving new entry should accepts no parameter from the url itself. so maybe have the id passed as part of the body as well?